pyiron_atomistics/calphy/job.py

```diff
@@ -640,6 +699,17 @@ def run_static(self):
         self.status.collect = True
         self.run()
 
+    def get_structure(self, iteration_step=-1):
+        """
+        """
+        if not iteration_step in [0, -1]:
+            raise ValueError("get_structure for calphy can only give frame 0 or -1")
+        if iteration_step == 0:
+            return self.structure
+        elif iteration_step == -1:
+            return self.output.structure_final
+
+
```

Comment:
Implement `HasStructure` instead of providing this method directly.

Reply:
@pmrv Could you please point me to an example where it is used?

Reply:
Here are some example from [StructureContainer](https://github.com/pyiron/pyiron_atomistics/blob/833eb7b0d76590bd09058e1fd28192cba82a0d79/pyiron_atomistics/atomistics/job/structurecontainer.py#L134) and [StructureStorage](https://github.com/pyiron/pyiron_atomistics/blob/833eb7b0d76590bd09058e1fd28192cba82a0d79/pyiron_atomistics/atomistics/structure/structurestorage.py#L224).  

Here's also the [docs](https://pyiron-atomistics.readthedocs.io/en/latest/apidoc/pyiron_atomistics.atomistics.structure.has_structure.html?highlight=HasStructure#pyiron_atomistics.atomistics.structure.has_structure.HasStructure).

Reply:
Thanks!

Reply:
Implemented this.